backend/tests/conftest.py

```diff
@@ -11,3 +11,8 @@ def app():
 @pytest.fixture
 def client(app):
     return app.test_client()
+
+
+@pytest.fixture()
+def runner(app):
```

Comment:
Using `cli_runner` will be more accurate.

Reply:
I delete this function and directly call db.create_all in test case.
[link](https://github.com/bamboo-cms/bamboo/pull/13/files#diff-e21ec187041c03f1c26c25c86d13f573099138a23fc65dc11bf6004744fc465dR9)